code/extensions/MultiFormObjectDecorator.php

```diff
@@ -14,43 +14,45 @@
  *
  * @package multiform
  */
-class MultiFormObjectDecorator extends DataExtension {
+class MultiFormObjectDecorator extends DataExtension
+{
+    private static $db = array(
+        'MultiFormIsTemporary' => 'Boolean',
+    );
 
-	private static $db = array(
-		'MultiFormIsTemporary' => 'Boolean',
-	);
+    private static $has_one = array(
+        'MultiFormSession' => 'MultiFormSession',
+    );
 
-	private static $has_one = array(
-		'MultiFormSession' => 'MultiFormSession',
-	);
+    public function augmentSQL(SQLSelect $query)
```

Comment:
This will need to be reverted back to 3.x compatible syntax

```
 public function augmentSQL(SQLQuery $query)
 {
 $where = $query->getWhere();
 }
 ```